src/nodes.coffee

```diff
@@ -3204,6 +3234,13 @@ exports.Splat = class Splat extends Base
 
   unwrap: -> @name
 
+  astType: -> 'SpreadElement'
+
+  astProperties: -> {
+    argument: @name.ast()
+    @postfix
```

Comment:
`postfix` is an extension to the Babel-style `SpreadElement` AST

On a related note, you've mentioned in previous code reviews that we prefer only using postfix `...` in the Coffeescript codebase. In the ESLint integration, I created a `spread-direction` rule (which this exposing of `postfix` in the AST made super easy) that enforces such a preference. So when we get to the point of setting that up in this repo, we can turn on that rule and have ESLint do the enforcement for us

Reply:
The left-hand-side `...` didn’t exist until CoffeeScript 2, when it was added as an equivalent option to the `...` that’s been around for years. Hence the preference for `...`, because naturally that’s what the CoffeeScript compiler codebase itself uses since it predates CoffeeScript 2.

There are definitely times for the left side version though, such as when using Vue:

```coffee
export default
  name: 'my-component'

  computed: {
    ...mapGetters 'user', ['profile']
  }
```

Without the option for the left-hand `...`, the `mapGetters` here would need to be written as `mapGetters('user', ['profile'])...` which isn’t terrible, but less elegant than parentheses-less; and less idiomatic with JS Vue code.